roadmap/implementers-guide/src/runtime/scheduler.md

```diff
@@ -175,13 +175,14 @@ Actions:
 1. Set `SessionStartBlock` to current block number.
 1. Clear all `Some` members of `AvailabilityCores`. Return all parathread claims to queue with retries un-incremented.
 1. Set `configuration = Configuration::configuration()` (see [`HostConfiguration`](../types/runtime.md#host-configuration))
-1. Resize `AvailabilityCores` to have length `Paras::parachains().len() + configuration.parathread_cores with all `None` entries.
+1. Determine the number of cores & validator groups as `n_cores`. This is the maximum of
+  1. `Paras::parachains().len() + configuration.parathread_cores`
+  1. `n_validators / max_validators_per_core` > N if `configuration.max_validators_per_core` is `Some` and non-zero.
```

Comment:
What is `N` here? And I am not clear why do we have `>` here.

Reply:
Vestiges from a previous edit I made. Thanks for catching it